app/views/admin/schools/notes/index.html.erb

```diff
@@ -1,34 +1,19 @@
-<div class="d-flex justify-content-between align-items-center">
-  <h1><%= @school.name %> Notes & Issues</h1>
-  <div>
-    <%= header_nav_link 'School group', admin_school_group_url(@school.school_group) if @school.school_group %>
-  </div>
-</div>
+<%= render 'header', title: "#{@school.name} Notes & Issues" do %>
+  <%= header_nav_link 'School group', admin_school_group_url(@school.school_group) if @school.school_group %>
+<% end %>
+
 <p>
-  <%= link_to 'New Note', new_admin_school_note_path(@school), class: 'btn btn-primary' %>
-  <%= link_to 'New Issue', new_admin_school_note_path(@school, note_type: :issue), class: 'btn btn-primary' %>
+  <%= link_to "#{((fa_icon('sticky-note text-secondary')))} New Note".html_safe, new_admin_school_note_path(@school, note_type: :note), class: 'btn btn-primary btn-sm' %>
```

Comment:
Are the multiple brackets around the fa_icon required?

Reply:
Hmm no, not sure how that happened. Thanks for picking that up!